### PR TITLE
feat: Add new Badges

### PR DESCRIPTION
**Pull request by Crash--**
Follow up of https://github.com/cozy/cozy-ui/pull/1390

We decided that it was more a Badge than an IconStack. 

I created a container of 24x24 following @joel-costa 's advice. In this container, we have a medium circle or a small circle 

https://crash--.github.io/cozy-ui/react/#/Basics?id=badge

### CODE REVIEW

File: react/Badge/Readme.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 `<Badge />` component wraps the element on which you want to apply a badge. In the following examples, it is applied to compact `<ButtonAction />`.
 
+You should use `<Badge />` when we want to add additionnal informations on the wrapped component
```
Comment (y-lohse) [inline react/Badge/Readme.md]:
```suggestion
You should use `<Badge />` to provide additional information related to the wrapped content.
```

Just a suggestion, both are correct but I don't like the you/we switch :)

File: react/Badge/Readme.md [hunk 2]
```diff
@@ -28,3 +30,26 @@ import ButtonAction from 'cozy-ui/transpiled/react/ButtonAction';
   </p>
 </div>
 ```
+
+
+#### Alignment
+
+Only support `bottom-right`
```
Comment (y-lohse) [inline react/Badge/Readme.md]:
```suggestion
Only supports `bottom-right` 
```

File: react/Badge/Readme.md [hunk 3]
```diff
@@ -28,3 +30,26 @@ import ButtonAction from 'cozy-ui/transpiled/react/ButtonAction';
...
+import Badge from 'cozy-ui/transpiled/react/Badge'
+import Icon from 'cozy-ui/transpiled/react/Icon';
+
+
+
+<div>
+  <p>
+    <Badge 
+      content={ <Icon icon="link" size={10} />} type="normal" alignment="bottom-right" size='medium'>
+      <Icon icon="cloud" size="32" />
```
Comment (y-lohse) [inline react/Badge/Readme.md]: Could we use a `ButtonAction ` here as well? I find it's a bit easier to understand which one the badge is and which one the other component is if we don't use Icon for both :)

Comment (Crash--) [reply react/Badge/Readme.md]: It was to have a really life example since we currently don't use Badge with a ButtonAction

Comment (y-lohse) [reply react/Badge/Readme.md]: Mmh ok fair enough!

File: react/Badge/Readme.md [hunk 4]
```diff
@@ -28,3 +30,26 @@ import ButtonAction from 'cozy-ui/transpiled/react/ButtonAction';
...
+    </Badge>
+  </p>
+  
+</div>
+```
```
Comment (ptbrowne) [inline react/Badge/Readme.md]: I am not sure why we add doc for `alignment` since it can only have one value ?

Comment (Crash--) [reply react/Badge/Readme.md]: For now, but we plan to add top-right pretty soon

**Pull request merged**